core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -625,7 +625,7 @@ private boolean restoreInternal(Path rootDir, PageStoreOptions options) {
 
   @Override
   public List<PageId> getCachedPageIdsByFileId(String fileId, long fileLength) {
-    int numOfPages = (int) (fileLength / mPageSize);
+    int numOfPages = (int) ((fileLength - 1) / mPageSize) + 1; //ceiling round the result
```

Comment:
Why not directly use ceiling? `ceil((double)fileLength / mPageSize)`? Would that give wrong result in corner cases?

Reply:
Yes, the ceil does not work for the empty file (size is 0).  this change is for the local cache running on hdfs data node, so there might be some empty blocks